cli/cli.go

```diff
@@ -284,6 +286,17 @@ func fetchIndex() {
 	}
 }
 
+func installFromVersionFile() bool {
+	version, err := ioutil.ReadFile(".ruby-version")
+	if err != nil {
+		return false
+	}
+
+	fmtc.Println("Installing version %s from .ruby-version", version)
+	installCommand(string(version))
```

Comment:
You got error if `.ruby-version` contains spaces or newline symbols. I think you must handle this situations.